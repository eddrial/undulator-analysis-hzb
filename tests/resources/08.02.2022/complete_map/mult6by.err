 error report for the data set mult6by.dvm         
--------------------------------------------
      
         128   data points thrown away to get a
         monotonically increasing data set
 the truncated data set has only   1591 data points 
        out of totally   2557 data points
 the rms value of the residuals is  0.44762E-02
 the maximum residual is  0.13412E-01
 the rms value of the lambda/2-filtered data is  0.33980E-01
 the maximum value of the lambda/2-filtered data is  0.25908E+00
