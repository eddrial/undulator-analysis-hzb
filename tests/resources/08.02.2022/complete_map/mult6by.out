mult6by.dvm 
            

 names of parameters as follows : 
b(1) == amplitude, 1st harm.
b(2) == amplitude, 3rd harm.
b(3) == amplitude, 5rd harm.
....                        
b(n) == amplitude, n^th harm.
b(n+1) == lambda, mm.
b(n+2) == x0, 1st harm, mm.
b(n+3) == x0, 3rd harm, mm.
b(n+4) == x0, 5th harm, mm.
....
b(nparm) == x0, n^th harm, mm.

 parameters:
   1  9.6258020E-01
   2 -9.5661823E-03
   3 -9.6195079E-03
   4 -1.5671467E-04
   5  2.3564335E-04
   6  5.5982536E+01
   7  1.0438690E+03
   8  1.0436388E+03
   9  1.0439473E+03
  10  1.3070126E-01
  11 -7.2743225E+00
 number of iterations:    7
     sumsq:  3.18787E-02
       rms:  4.47626E-03  dimension of magnetic field
       rms:  4.65024E-01  percent of b_eff
   rms_ext:  4.04773E-01  percent of b_eff
 B_eff =   9.6259E-01
 B_max =   9.4382E-01
 K_eff =    5.035
 lamda_1 (nm) =     34.73629

        eigenvectors in parameter space

 error                         direction
     1.29,  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  1.00  0.00
     0.67,  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  1.00
     0.05,  0.00  0.00  0.00  0.00  0.00  0.00  0.00  1.00  0.00  0.00  0.00
     0.03,  0.00  0.00  0.00  0.00  0.00  0.00  0.00  0.00  1.00  0.00  0.00
     0.00,  0.00  0.00  0.00  0.00  0.00 -0.05  1.00  0.00  0.00  0.00  0.00
     0.00, -0.53  0.47  0.17  0.21 -0.07  0.65  0.03  0.00  0.00  0.00  0.00
     0.00,  0.33  0.44  0.55 -0.53  0.32  0.01  0.00  0.00  0.00  0.00  0.00
     0.00,  0.63  0.48 -0.07  0.54 -0.28 -0.01  0.00  0.00  0.00  0.00  0.00
     0.00,  0.08  0.15 -0.53  0.12  0.81  0.14  0.01  0.00  0.00  0.00  0.00
     0.00, -0.22 -0.08  0.54  0.58  0.38 -0.41 -0.02  0.00  0.00  0.00  0.00
     0.00,  0.39 -0.58  0.29  0.16  0.13  0.62  0.03  0.00  0.00  0.00  0.00
   
  calculated kicks for trajectory compensation: 
 kick_1:  -0.05059 Tmm
 kick_2:   0.05059 Tmm
      
 the rms value of the second derivative is  0.00000E+00
 the maximum value of the second derivative is  0.00000E+00
      
 the maximum transverse movement of the electron in mu_meter
                  (peak to peak) is  0.35792E+02
      
 the truncated data set has only   1591 data points 
      
 the rms value of the residuals is  0.44762E-02
 the maximum residual is  0.13412E-01
      
 the rms value of the lambda/2-filtered data is  0.33980E-01
 the maximum value of the lambda/2-filtered data is  0.25908E+00
      
 the rms phase error (all data) is    4.875 deg.
 the maximum phase error (all data) is   12.700 deg.
      
 the rms phase error (extrema) is    6.172 deg.
 the maximum phase error (extrema) is   11.657 deg.
      
 the Walker-rms phase error (all data) is   19.497 deg.
 the Walker-maximum phase error (all data) is   38.083 deg.
      
 the Walker-rms phase error (extrema) is    5.052 deg.
 the Walker-maximum phase error (extrema) is   11.544 deg.
  
  phase: straight line subtracted
 -----------------------------------------
 total phase error             4.875418    
 local phase error             2.361502    
 slowly varying phase error    4.121576    
 oscillatory phase error       1.098007    
 -----------------------------------------
